app/controllers/admin_controller.rb

```diff
@@ -5,5 +5,14 @@ def require_admin
     verboten! unless current_user.super_admin?
   end
 
-  def dashboard; end
+  def dashboard
+    @recent_organizations = Organization.where('created_at > ?', 1.week.ago)
+    @recent_users = User.where('created_at > ?', 1.week.ago)
+    @active_users = User.where('last_sign_in_at > ?', 1.week.ago)
```

Comment:
@seanmarcia This is the "previous" sign in date, not the "current" one. You want `current_sign_in_at` for their most recent sign in date. 

600 minutes is 10 hours. You're likely going to log people out every day for this "nice to have". I'm not sure it's a good idea to do that in order to have an admin "active users" list.